### ISSUE TITLE
Wondering if the proxy nca is just the softmax loss

### ISSUE THREAD

**Issue by SupetZYK** [dichotomies/proxy-nca#14]
In the experiment of proxy NCA, the number of proxies is set to the number of classes of training set. So what is the difference between proxy NCA and pure softmax loss?

**Comment by dichotomies** [comment]
They are related, but not the same. See `https://arxiv.org/pdf/1909.05235.pdf`, p. 4, remark 3 for a detailed comparison between both losses.
